tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>As far as I can tell, you don't need:</p><ul><li><code>pprint</code></li><li><code>Chadoq2</code></li><li><code>Image</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>